src/lib/key_gen/sample_keypairs.ml

```diff
@@ -0,0 +1,43 @@
+open Core_kernel
+open Signature_lib
+
+let kps =
+  lazy
+    (let n = 1200 in
```

Comment:
This number 1200 feels pretty aggressively high. It could even quite naturally be a parameter where this is turned into a function, so that we only generate keys that we actually need for the unit tests.

Reply:
@mrmr1993 what do you think of just placing these `1200` keys as plain text files to the repository?

Actually, there aren't too many usages. Maybe we could check the code to determine the max number needed (maybe it's just 30?).

Reply:
Having it be dynamic gives us more flexibility in the integration tests (which is the only place they're used that I could find). That said, this is already an improvement.